tests/steps/steps.py

```diff
@@ -6,19 +6,39 @@
 
 @given("Account is created and added to the database")
 def step_impl(context):
-    pass  # Placeholder for account creation
+    url = BASE_URL + "accounts/"
+    data ={
+        "initial_balance": 2000,
+        "owner": "Greg"
+    }
+    response = requests.post(url, json=data)
+    json_data = response.json()
+    context.status_code = response.status_code
+    context.id = json_data["id"]
+    context.balance = json_data["balance"]
 
 
 @when('I call "{endpoint_name}" endpoint and store response')
 def step_impl(context, endpoint_name):
-    pass  # Placeholder for calling the specified endpoint and storing the response
+    url = f"{BASE_URL}accounts/{context.id}"
+    response = requests.get(url)
+    json_data = response.json()
```

Comment:
same here, create helper function
def get_account(account_id):
    url = f"{BASE_URL}accounts/{account_id}"
    response = requests.get(url)
    return response

and call it inside of the step

@when('I call "{endpoint_name}" endpoint and store response')
def step_impl(context, endpoint_name):
    response = get_account(context.id)
    json_data = response.json()
    context.status_code = response.status_code
    context.received_id = json_data["id"]
    context.received_balance = json_data["balance"]